commit ddb97770715fed585f1e69cb5478ff5789b5c70b (refs/changes/12/3812/2)
Author: Cody Peter Mello <cody.mello@joyent.com>
Date:   2018-04-12T00:12:12+00:00 (1 year, 6 months ago)
    
    TRITON-317 Copy SVGs into FWAPI documentation
    Reviewed by: Trent Mick <trentm@gmail.com>
    Approved by: Trent Mick <trentm@gmail.com>

diff --git a/.gitignore b/.gitignore
index 6798ae8..9475b3a 100644
--- a/.gitignore
+++ b/.gitignore
@@ -8,6 +8,7 @@ docs/*.json
 docs/*.html
 docs/examples.md
 docs/rules.md
+docs/media/img/*.svg
 cscope.in.out
 cscope.po.out
 cscope.out
diff --git a/Makefile b/Makefile
index a151103..a8d6a39 100644
--- a/Makefile
+++ b/Makefile
@@ -102,6 +102,7 @@ node_modules/fwrule/docs/examples.md: node_modules/fwrule
 docs/rules.md: node_modules/fwrule/docs/rules.md
 	$(TOP)/tools/restdown-header "Firewall API Rule Syntax" > docs/rules.md
 	cat node_modules/fwrule/docs/rules.md >> docs/rules.md
+	cp node_modules/fwrule/docs/media/img/*.svg docs/media/img/
 
 docs/examples.md: node_modules/fwrule/docs/examples.md
 	$(TOP)/tools/restdown-header "Firewall API Examples" > docs/examples.md
